### DISCUSSION TITLE
How to increase the Oryx build timeout?

### DISCUSSION THREAD

**Discussion by wesoos** [Azure/static-web-apps#669] [Q&A] [answered]
Hello, I have a react app I'm deploying via Github Actions.  During the build step, Oryx times out after 15 minutes.  Is there a way to increase the timeout?

Thanks

**Comment by miwebst** [comment, marked as answer]
Hey @wesoos, this can either be indicative of a timeout or an Oryx issue. Here is an example of how to bump the timeout: https://github.com/Azure/static-web-apps/issues/113

If your app does not usually take more than 15 minutes it could be an Oryx issue and bumping the timeout won't help.
